projects/packages/a8c-mc-stats/changelog/try-remove-composer-update

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
I'd call it patch, not minor. It's not changing anything about the code itself, just how some of the tests get run.

Same everywhere else.

Reply:
Fixed